### PR TITLE
update french translation

### PR DESCRIPTION
**Pull request by Martronic-SA**

### CODE REVIEW

**Comment by rnixx** [comment]
Thanks

File: src/bda/plone/shop/locales/fr/LC_MESSAGES/bda.plone.shop.po [hunk 1]
```diff
@@ -46,7 +46,7 @@ msgstr ""
 #. Default: "Cart"
 #: ./interfaces.py:117
 msgid "Cart"
-msgstr ""
+msgstr "Cart"
```
Comment (agitator) [inline src/bda/plone/shop/locales/fr/LC_MESSAGES/bda.plone.shop.po]: shouldn't that be "Panier"?

Comment (Martronic-SA) [inline src/bda/plone/shop/locales/fr/LC_MESSAGES/bda.plone.shop.po]: Sure!

**Comment by Martronic-SA** [comment]
There are some confusions with Gross (HT?? no taxes) and Net (TTC?? taxes included)

#. Default: "Show price with taxes included"
#: ./at.py:185
#: ./dx.py:110
msgid "help_item_display_gross"
msgstr "Afficher les prix TTC"

**Comment by rnixx** [comment]
net is without VAT, gross including VAT, or what do you mean?

**Comment by Martronic-SA** [comment]
On the piece of *.po above, msgid is "help_item_display_**gross**" and the default is "Show price **taxes included**"
And Gross or Net could also mean "Before discounts" and "all discounts applied"

**Comment by rnixx** [comment]
gross and net in our case always means with or without value added tax

**Comment by rnixx** [comment]
And what about this one now? BTW still "Cart" here. Why have you opened 2 PR's for the same task?

**Pull request merged**